app/views/welcome/index.html.erb

```diff
@@ -13,3 +13,10 @@
     <p>You're logged in <%= current_user.name%>!</p>
   </section>
 <% end %>
+
+<h3>Check Out All The Images We Have So Far:</h3>
+<% @images.each do |image| %>
+  <% if !image.image.nil? %>
+    <%= image_tag(image.image, style:'width:10%') if image.image.attached? %>
+  <% end %>
+<% end %>
```

Comment:
A little ineloquent, so in a future refactor, make it so you can't add an image without actually attaching an image.